socs/agent/smurf_recorder.py

```diff
@@ -21,6 +23,19 @@ class FlowControl(Enum):
     CLEANSE = 3
 
 
+def check_port(host, port, timeout=10):
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.settimeout(timeout)
+    s.setblocking(1)
+    try:
+        s.connect((host, port))
+        return True
+    except socket.error as error:
```

Comment:
`error` is assigned here but never used. Would it be useful to log that error if it arises?

Reply:
I don't think it's especially useful, and I'm not super inclined to add a debug statement since there's no log instance in scope. If you really think I should log something we can, but for now I'll just remove the error assignment.